pkg/cmd/admin/groups/new.go

```diff
@@ -90,7 +90,7 @@ func (o *NewGroupOptions) Complete(f *clientcmd.Factory, cmd *cobra.Command, arg
 	templateFile := kcmdutil.GetFlagString(cmd, "template")
 	noHeaders := kcmdutil.GetFlagBool(cmd, "no-headers")
 	allowMissingTemplateKeys := kcmdutil.GetFlagBool(cmd, "allow-missing-template-keys")
-	printer, _, err := kubectl.GetPrinter(outputFormat, templateFile, noHeaders, allowMissingTemplateKeys)
+	printer, _, err := printers.GetStandardPrinter(outputFormat, templateFile, noHeaders, allowMissingTemplateKeys, nil, nil, nil)
```

Comment:
@ncdc do you know what to pass here? The copier is the scheme. Which RESTMapper makes sense and which codecs?